html/elements/link.json

```diff
@@ -751,7 +751,7 @@
             "__compat": {
               "description": "rel=compression-dictionary",
               "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Reference/Attributes/rel#compression-dictionary",
-              "spec_url": "https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-compression-dictionary#name-link-relation-registration",
+              "spec_url": "https://httpwg.org/http-extensions/draft-ietf-httpbis-compression-dictionary.html#name-link-relation-registration",
```

Comment:
Same issue as above, actually.

These are two versions of the spec, but only IDS for https://httpwg.org/http-extensions/draft-ietf-httpbis-compression-dictionary.html are in the ID list.